spec/lib/rex/proto/kerberos/credential_cache/krb5_ccache_presenter_spec.rb

```diff
@@ -293,5 +334,64 @@
         EOF
       end
     end
+
+    describe '#print_bindata_model' do
+
+      context 'when the bindata object has fields that are not 1bit' do
+        it 'should raise an exception' do
+          binary_value = [7].pack('C')
+          dummy_attributes = DummyAttributes.read(binary_value)
+          expect { subject.print_bindata_model(dummy_attributes, bit_length: 32) }.to raise_error RuntimeError, 'Unsupported field type BinData::Bit2 - expected BinData::Bit1'
+        end
+      end
+
+      context 'when a bit length less than the number of flags is passed' do # better description
+        it 'should raise an exception' do
+          binary_value = [7].pack('N')
+          group_attributes = Rex::Proto::Kerberos::Pac::GroupAttributes.read(binary_value)
+          expect { subject.print_bindata_model(group_attributes, bit_length: 2) }.to raise_error RuntimeError, 'Bit length(2) should not be less that the number of flags(32) within the BinData model'
+        end
+      end
+
+      context 'with binary value' do
+        it 'should return formatted flag descriptions for GROUP_MEMBERSHIP flags with RESOURCE set' do
+          binary_value = [536870912].pack('N')
+          group_attributes = Rex::Proto::Kerberos::Pac::GroupAttributes.read(binary_value)
+          expect(subject.print_bindata_model(group_attributes, bit_length: 32)).to eq <<-EOF.rstrip
```

Comment:
should be able to update the indentation on the left if you use `to match_table <<~TABLE` instead